### DISCUSSION TITLE
Can I configure prisma not to use prepare statement?

### DISCUSSION THREAD

**Discussion by notzheng** [prisma/prisma#5479] [Q&A]
When I use prisma , I got an error 

```
ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(Server(ServerError { code: 1047, message: "Prepare unsupported!", state: "HY000" })) })
```

Because my database administrator disabled the prepare statement, Can I configure prisma not to use prepare statement?

**Comment by ryands17** [comment]
@notzheng 👋 

Currently it's not possible to prevent Prisma from using a prepared statement, so I would suggest creating a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for the same so that we can look into this :)

**Reply by notzheng** [reply]
Thanks, it's here: https://github.com/prisma/prisma/issues/5510

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
